bindings/apple/README.md

```diff
@@ -6,24 +6,28 @@ This project and build script demonstrate how to create an XCFramework that can
 
 * the Rust toolchain
 * UniFFI - `cargo install uniffi_bindgen`
```

Comment:
Is this step still required, or can it be removed now?

Reply:
From what I understand the xtask should use the bindgen package that is linked against the sdk, so no, I don't think this is required anymore